app/controllers/concerns/sufia/batch_uploads_controller_behavior.rb

```diff
@@ -5,7 +5,6 @@ module BatchUploadsControllerBehavior
     include CurationConcerns::CurationConcernController
 
     included do
-      layout "sufia-one-column"
```

Comment:
Does this controller behavior not need `with_themed_layout '1_column'` or is that accounted for elsewhere?